### PR TITLE
Zjet analysis changes

### PR DESCRIPTION
**Pull request by HerrHorizontal**
This introduces config and code changes for the Z+Jet analysis. This should not be in conflict with the jet calibration.

### CODE REVIEW

**Review by cverstege — APPROVED** [review]
I think it's ready to merge

File: cfg/python/defaultconfig.py [hunk 1]
```diff
@@ -528,10 +528,10 @@ def mm(cfg, **kwargs):
         'filter:LeadingJetEtaCut',
         'filter:MinNMuonsCut',
         'filter:MaxNMuonsCut',
+        'filter:ValidZCut', # includes Z mass cut
```
Comment (HerrHorizontal) [inline cfg/python/defaultconfig.py:531]: Does it make sense to require a valid Z, before applying the Muon-cuts?

**Review by HerrHorizontal — COMMENTED** [review]
Single comment:

File: cfg/python/defaultconfig.py (hunk 1 above)
Comment (HerrHorizontal) [reply cfg/python/defaultconfig.py:531]: Muon cuts run only on muons assigned to a Z-candidate. So it makes no improvements to run the Muon-cuts before the ValidZ-cut.

Comment (cverstege) [reply cfg/python/defaultconfig.py:531]: Processing root://cmsxrootd-redirectors.gridka.de//store/user/cverstege/Skimming/ZJet_RunIISummer20UL17MiniAODv2_DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8/job_9_output.root ...
Status: 21000/84726 - 00:01:12/00:01:36 - 879 / s -  25%    FATAL: No valid Z! Please check if the ValidZ cut was already applied.
WARN : Aborting application. Reason: Fatal log at [/work/cverstege/ZJet/CMSSW_10_6_30_ULRun2/src/Excalibur/Compile/interface/Filters/ZJetCutsFilter.h:232]
zjet run with config 2017ul_DYJetsToLL_MLM_JEC_UP FAILED!

**Pull request merged**